package.json

```diff
@@ -11,6 +11,12 @@
   "devDependencies": {
     "husky": "^3.1.0",
     "lerna": "^3.19.0",
-    "lerna-changelog": "^0.8.3"
+    "lerna-changelog": "^0.8.3",
+    "semver": "^6.3.0",
+    "shelljs": "^0.8.3",
+    "vue": "^2.6.4",
+    "vue-template-compiler": "^2.5.0",
```

Comment:
Check the dependencies in individual packages, some of this might be needed at runtime.

Reply:
Could you elaborate? vue and the template compiler are only there to satisfy vuetify-loader's peer dependencies.

Reply:
Maybe not specifically vue-template-compiler, but shelljs is needed at runtime: #147

Reply:
In the package it's used, not the root